### PR TITLE
[7] `Pool` and `CLPool` fee initialization

### PR DESCRIPTION
**Pull request by FonsSicca**
Implements:
- Correctly initialisation of `baseFee` and `currentFee` for non-CL and CL Pools. New events were included (e.g. `TickSpacingEnabled`)
- Renaming of some files

## Summary by CodeRabbit

* **New Features**
  * Added tick-spacing fee events and a fee-to-tick-spacing mapping; pools now initialize and persist baseFee and currentFee.

* **Behavior**
  * Default fees applied for volatile and stable pools at creation; dynamic fee fields updated as events arrive.

* **Configuration**
  * Standardized swap module names across networks and added missing module entries.

* **Tests**
  * Expanded coverage for mapping, initialization, updates, timestamps, multi-mapping and edge cases.

<sub>✏️ Tip: You can customize this high-level summary in your review settings.</sub>

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -32,6 +32,7 @@ contracts:
     handler: src/EventHandlers/CLFactory.ts
     events:
       - event: PoolCreated
+      - event: TickSpacingEnabled
```
Comment (FonsSicca) [inline config.yaml]: new event that allows the mapping of `tickSpacing` <-> `fee`

This mapping can be modified by `SetCustomFee` or `CustomFeeSet` from, respectively, `CustomSwapFeeModule` and `DynamicSwapFeeModule` contracts

File: src/EventHandlers/SwapFeeModule/DynamicSwapFeeModule.ts
Comment (FonsSicca) [inline src/EventHandlers/SwapFeeModule/DynamicSwapFeeModule.ts]: renaming
